packages/react/src/components/MultiSelect/FilterableMultiSelect.tsx

```diff
@@ -737,6 +754,11 @@ const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect<
           event.stopPropagation();
         }
 
+        if (readOnly) {
+          event.preventDefault();
+          return;
+        }
+
```

Comment:
I don't think code will reach here if we are using `readOnlyEventHandlers`.

Reply:
removed